chatbees/client_models/collection.py

```diff
@@ -154,6 +203,7 @@ def chat(self, doc_name: str = None) -> Chat:
             doc_name=doc_name
         )
 
+    # deprecated. Please use create_website_ingestion function
```

Comment:
let's deprecate later. get_ingestion does not return the PageStats yet.

Reply:
sg, removed this line